packages/components/src/__actions__/Button/v3/subcomponents/PendingContent/PendingContent.module.css

```diff
@@ -0,0 +1,16 @@
+.pendingContent {
+  display: inline-flex;
+  align-items: center;
+  gap: var(--pending-gap, var(--spacing-4));
```

Comment:
To assist with identifying the source (sorry it's long - open to suggestions)
```suggestion
  gap: var(--button-pending-content-gap, var(--spacing-4));
```

Reply:
Makes sense - it's long but this is pretty specific